docs/configuration/secrets/troubleshoot.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the secrets used by Codacy.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
```

Comment:
This is a more generic comment, but I'm thinking we should check with Support if they're comfortable whenever we suggest to the users that they should use support@codacy.com to contact Codacy. At the very least, Support should know about these "entry points" so they are empowered to tackle the tickets from customers more efficiently. (@IGFCoimbra, @paulacodacy).

Reply:
As we prefer, I have no strong opinion about this.
I was just putting this were it makes sense for our customers to know where to go for help.